_rules/link-non-empty-accessible-name-c487ae.md

```diff
@@ -274,6 +283,14 @@ This `a` element has an [explicit role][] of `none`. However, it is [focusable][
 <a href="https://www.w3.org/WAI" role="none"> </a>
 ```
 
+#### Failed Example 11
+
+This `span` element has an [inheriting semantic role][] of `link` and an empty [accessible name].
+
+```html
+See [<a onclick="location.href='https://act-rules.github.io/'" role="doc-biblioref"></a>]
```

Comment:
```suggestion
See [<span onclick="location.href='https://act-rules.github.io/'" role="doc-biblioref"></span>].
```

Reply:
Maybe adding an image inside the `span` here. Visually, nothing is displayed and there is almost nowhere to actually click, which is at least weird.
Adding [this image](https://github.com/act-rules/act-rules.github.io/blob/develop/test-assets/shared/act-logo.png) with an empty `alt` would be better, I think.